src/components/link/EnvironmentVariables.js

```diff
@@ -0,0 +1,30 @@
+import Link from "next/link";
+
+export const getLinkData = (environmentSlug, projectSlug) => ({
+  urlObject: {
+    pathname: "/projects/[projectSlug]/[environmentSlug]/environment-variables",
```

Comment:
`pathname` should just be `/environment-variables`, just how it's written in server.js:
 `app.render(req, res, '/environment-variables', ...);`
so Next knows whats the actual component for the path and displays it as written in `asPath`, otherwise it will actually do a full reload/re-render and there will be a "signing you in" and "loading" popups after navigating.